File: ReSharper.FSharp/test/data/common/assemblyReaderShim/Method - Extension 01.fs
module Module

open System.Collections.Generic

Class.StringExt("")
"".StringExt()

Class.ObjSeq([1])
[obj()].ObjSeq()
[1].ObjSeq()


Class.GenericSeqExt<int>([1])
[1].GenericSeqExt<int>()

Class.GenericSeqExt<string>([""])
[""].GenericSeqExt<string>()

[""].GenericSeqExt<int>()
[1].GenericSeqExt<string>()

Class.StringSeqExt([""])
[""].StringSeqExt()

Class.StringSeqExt([1])
[1].StringSeqExt()

File: ReSharper.FSharp/test/data/common/assemblyReaderShim/Method - Extension 01.cs
using System.Collections.Generic;

public static class Class
{
    public static void StringExt(this string s)
    {
    }

    public static void ObjSeq(this IEnumerable<object> e)
    {
    }
    
    public static void GenericSeqExt<T>(this IEnumerable<T> e)
    {
    }
    
    public static void StringSeqExt(this IEnumerable<string> e)
    {
    }

}
